### DISCUSSION TITLE
Compile issues on Fedora 40.

### DISCUSSION THREAD

**Discussion by kevitivity** [hmatuschek/qdmr#433] [General]
Hello and thank you for this project.

I'm trying to compile from source on Fedora 40. First issue I'm having is that there is no libusb.h.

dnf reports: Error: Unable to find a match: libusb-devel

**Comment by PeterSprague** [comment]
try libusb1-devel

Maybe the instructions for Fedora need to be corrected

**Reply by kevitivity** [reply]
Thank you. That did the job in terms of getting it to compile on Fedora 40.

I also needed to do export LD_LIBRARY_PATH=/usr/local/lib64:$LD_LIBRARY_PATH to get it to run.

Otherwise, I got this error:

qdmr: error while loading shared libraries: libdmrconf.so.0.11: cannot open shared object file: No such file or directory

Thanks again for your work on this!
